decidim-core/spec/lib/events/base_event_spec.rb

```diff
@@ -49,7 +49,7 @@ module Decidim
       describe "when the resource_text is a translation hash" do
         let(:resource_text) do
           {
-            en: "My string",
+            :en => "My string",
```

Comment:
I think this could be converted to new hash style.

Reply:
Done in 984ace2000ad97e768bb8eba091d4ba7eda2ac11